data.py

```diff
@@ -194,28 +195,28 @@ def load_query_string(self, data: str) -> None:
 
 class DataV3(Data):
     @staticmethod
-    def get_topics() -> List[str]:
+    def get_topics() -> List[topics.Topic]:
         return [
-            str(topics.SensorModules.all_sensors),
-            str(topics.DAShboard.overlay_message),
-            str(topics.PowerModelV3.recommended_sp),
-            str(topics.PowerModelV3.predicted_max_speed),
-            str(topics.PowerModelV3.plan_name),
+            topics.WirelessModule.data(),
+            topics.Camera.overlay_message,
+            # TODO: BOOST currently publishes data in the deprecated V2 format
+            # on the V3 topic. Uncomment below when updated.
+            # topics.BOOST.recommended_sp,
+            # topics.BOOST.predicted_max_speed,
+            # TODO: Implement handling topics.BOOST.generate_complete
         ]
 
     def load_data(self, topic: str, data: str) -> None:
         """Update stored fields with data from a V3 sensor module data packet.
         """
-        if topics.DAShboard.overlay_message.matches(topic):
+        if topic == topics.Camera.overlay_message:
```

Comment:
Just out of curiosity, would it be better to use `is` instead of `==`? Since these two should point to the same object in memory, right?

Reply:
Object instance / memory dependencies are difficult to manage (which is why `is` is generally only used for comparisons with `null` or `undefined`) whereas in this case, it is a value dependency.
For example, there is no reason why it has to be the same value in memory as the comparison is about the topic string value.

Reply:
Makes sense, thanks for the explanation